collector/Dockerfile

```diff
@@ -58,7 +63,6 @@
 
 
 FROM python:2-onbuild
+ENV CLUSTER_INSIGHT_MODE minion
 
-EXPOSE 5555
```

Comment:
Yes - as long as we use the -p parameter / use the replication controller file (which specifies the same using containerPort/hostPort).